TeamMandate/Mandate.md

```diff
@@ -14,13 +14,11 @@ This will be done through an open collaborative approach that will further ESDC'
 
 - Briefing senior management on key organisational problems and recommended steps to address them
 - Orchestrate the IT ecosystem to bring strategies to completion
-- Permeate and enforce modern IT practices and processes 
-- Authority to merge working groups, or committees, within IITB
-- Conduct inwards facing analysis of ESDC's IT processes
-- Required approval for new project onboarding 
-- Provide recommendations on resource IT resource allocation 
+- Support teams to introduce modern IT practices and processes
+- Recommend key guiding digital investments principles and rules for project onboarding
```

Comment:
Change "Recommend" to "Set"
Set key guiding IT investments principles and rules for project onboarding